src/hooks/useNotificationTypes.tsx

```diff
@@ -1175,6 +1179,73 @@ export const notificationTypes: NotificationTypeConfig[] = [
       };
     },
   },
+  {
+    type: NotificationType.VipTelegram,
+    useTrigger: ({ trigger }) => {
+      const stringGetter = useStringGetter();
+      const isConnected = useAppSelector(getIsAccountConnected);
+      const equity = useAppSelector(getSubaccountEquity, shallowEqual);
+      const { data: vaultAccount } = useLoadedVaultAccount();
+      const vaultBalance = vaultAccount?.balanceUsdc;
+
```

Comment:
After following steps in https://github.com/dydxprotocol/v4-web/pull/1901/files/50fdb651b95914e94f9965d01573902e02f13b37#r2383062316

You can use `const { telegramVipUrl } = useUrlConfigs()` and return if it doesn't exist, otherwise pass it into the notification being triggered.

Reply:
The reason we want to go through the trouble of adding it to the env config is so third party deployers that fork this repo are not forced to use these links